templates/events-my-events.php

```diff
@@ -18,10 +18,68 @@
 $event_page_title = __( 'My Events', 'gp-translation-events' );
 gp_tmpl_load( 'events-header', get_defined_vars(), __DIR__ );
 ?>
-
+<div class="events-links-to-anchors">
+	<ul>
+		<?php if ( ! empty( $events_i_am_or_will_attend_query->events ) ) : ?>
+			<li><a href="#events-i-am-or-will-attend"><?php esc_html_e( 'Events I am or will be attending', 'gp-translation-events' ); ?></a></li>
```

Comment:
```suggestion
			<li><a href="#events-i-am-or-will-attend"><?php esc_html_e( 'Events joined', 'gp-translation-events' ); ?></a></li>
```

I would like to suggest that the header be changed to `Events joined` for sake of brevity. I understand that we will get the entire page redesigned soon , however for the list of events joined, we can have the title of past events in grey colour while the title of current or upcoming events stays the same. See example below,

<img width="1546" alt="image" src="https://github.com/WordPress/wporg-gp-translation-events/assets/2834132/7044caab-639a-4f7a-9909-4bdeabc52962">

Reply:
What I want to explain to the user with the title "Events I am or will be attending" is that in this list she can find only the events that she is attending now, or she will attend in the future. With the "Events Joined" I understand that it fits the past events, too.